ush/prepobs_makeprepbufr.sh

```diff
@@ -1744,6 +1744,7 @@ backup AFWA ACARS into PREPBUFR"
 
 set +x
 cat <<\EOFmpp > MP_PREPDATA
+#!/bin/ksh 
 { echo
 #!/bin/ksh
```

Comment:
I think the #!/bin/ksh at line 1749 does not need to be here; Jack's addition at 1747 is correct.